README.md

```diff
@@ -1,19 +1,47 @@
 # Google CloudEvents - Go
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types used with Google services.
+At this moment the following types are available:
 
-## Prerequisites
+| Package | Struct | Description |
+| ------------- | ------------- | ------------- |
+| auditv1  | AuditLogWrittenEvent | The event is triggered when a new Cloud Audit Log entry is written. |
+| pubsubv1  | MessagePublishedEvent | This event is triggered when a new Cloud Pub/Sub event is published to a topic. |
```

Comment:
- I think we want the fully qualified name:
  - i.e. `google_events_cloud_audit_v1`
- We might want underscores between the name and version.